guides/common/modules/proc_configuring-host-based-authentication-control.adoc

```diff
@@ -1,62 +1,111 @@
-[id="Configuring_Host_Based_Authentication_Control_{context}"]
-= Configuring host-based authentication control
+[id="Configuring_Host_Based_Access_Control_{context}"]
+= Configuring host-based access control for {FreeIPA} users logging in to {Project}
 
-HBAC rules define which machine within the domain a {FreeIPA} user is allowed to access.
-You can configure HBAC on the {FreeIPA} server to prevent selected users from accessing {ProjectServer}.
-With this approach, you can prevent {Project} from creating database entries for users that are not allowed to log in.
+In {FreeIPA}, host-based access control (HBAC) rules define which {FreeIPA} users can access which {FreeIPA} hosts and which {FreeIPA} services can be used to gain access.
```

Comment:
Very clear explanation of how HBAC works. Is it necessary to always specify IPA? There are four occurrences in one sentence. But I understand that removing it might lead to potential misunderstanding in different languages and cultures.

You can use host-based access control (HBAC) rules to manage access control within your {FreeIPA} domain. HBAC rules define